app/javascript/app/providers/sections-content-provider/sections-content-provider-actions.js

```diff
@@ -0,0 +1,42 @@
+import { createAction, createThunkAction } from 'redux-tools';
+import { INDOAPI } from 'services/api';
+
+export const fetchSectionsContentInit = createAction(
+  'fetchSectionsContentInit'
+);
+export const fetchSectionsContentReady = createAction(
+  'fetchSectionsContentReady'
+);
+export const fetchSectionsContentFail = createAction(
+  'fetchSectionsContentFail'
+);
+
+export const fetchSectionsContent = createThunkAction(
+  'fetchSectionsContent',
+  params => (dispatch, state) => {
+    const { SectionsContent } = state();
+    if (SectionsContent && SectionsContent.data && !SectionsContent.loading) {
+      dispatch(fetchSectionsContentInit());
+      INDOAPI
+        .get('section_content', params)
+        .then(data => {
+          if (data) {
+            const sectionsContentMapped = {};
+            data.forEach(section => {
+              sectionsContentMapped[section.slug] = {
+                title: section.title,
```

Comment:
Are we going to have more than title and description for each section? Labels, subtitles, ...